judge/user_translations.py

```diff
@@ -7,7 +7,6 @@
     _translations = {}
 
     def translation(language):
-        global _translations
```

Comment:
is this random deletion safe?

Reply:
Lint told me to delete it (and because we don't re-assign `_translations` I think it is).